src/main/_docs/integration-guide/integration-continuous-integration.md

```diff
@@ -65,23 +76,25 @@ CODENVY_GITHUB_WEBHOOK_WEBHOOK1_FACTORY2_ID=factory2Id
 ```
 
 **For BitBucket Server**
-Update the `codenvy.env` with `Bitbucket Server` webhooks properties:
+Update the `codenvy.env` with `Bitbucket Server` webhooks properties. One webhook can update more than one Factory. Webhooks are tied to connectors through Factory IDs. There can be multiple webhooks, i.e. a new group of webhook properties can be created. In this case, change identifier WEBHOOKID, which can be any string.
 
 ```text  
-CODENVY_BITBUCKET_SERVER_WEBHOOK1_REPOSITORY_URL=https://github.com/testrepo.git
-CODENVY_BITBUCKET_SERVER_WEBHOOK1_WEBHOOK1_FACTORY1_ID=factory1Id
-CODENVY_BITBUCKET_SERVER_WEBHOOK1_WEBHOOK1_FACTORY2_ID=factory2Id
+CODENVY_BITBUCKET_SERVER_WEBHOOK_WEBHOOKID_REPOSITORY_URL=https://github.com/eclipse/che
+CODENVY_BITBUCKET_SERVER_WEBHOOK_WEBHOOKID_FACTORY1_ID=hfdhfd749347hd64
+CODENVY_BITBUCKET_SERVER_WEBHOOK_WEBHOOKID_FACTORY2_ID=<factory id>
 ```
 
 #### Jenkins Connector 
-Update the `codenvy.env` with connectors properties. Note that you can rename "WEBHOOK1" with any identifer. The system will treat Git and Jenkins Connector variables with the same name as related so ensure you have both identically named for each install:
+Update the `codenvy.env` with connectors properties. Note that you can rename "CONNETOR1" with any identifer. The system will treat Git and Jenkins Connector variables with the same name as related so ensure you have both identically named for each install. The can be n connectors, i.e. integration for several Jenkins jobs can be set up: CONNECTOR2, CONNECTORx etc.:
 
 ```text  
-#CODENVY_JENKINS_CONNECTOR_CONNECTOR1_FACTORY_ID=<factory id>
-#CODENVY_JENKINS_CONNECTOR_CONNECTOR1_URL=<url>
-#CODENVY_JENKINS_CONNECTOR_CONNECTOR1_JOB_NAME=<job name>
+CODENVY_JENKINS_CONNECTOR_CONNECTOR1_FACTORY_ID=r6p0l1sfnwm99k94
```

Comment:
I decided to leave it alone and then explain what needs to be done to add another group of connector props